src/lib/airtable/deliveryrequests/getDeliveryRequestNeedFormatted.js

```diff
@@ -6,10 +6,18 @@
 const { fields } = require("~airtable/tables/requestsSchema");
 
 exports.getDeliveryRequestNeedFormatted = (services) => {
-  if (services === undefined) {
+  if (!services) {
     return "Not stated";
   }
-  const servicesString = services.join(", ");
+
+  let servicesString = "";
+
+  if (typeof services === "string") {
+    servicesString = services;
+  } else if (Array.isArray(services)) {
+    servicesString = services.join();
```

Comment:
does it need the (", ") here for the join, like it was before? or will it know to join with comma as the separator?

Reply:
oh good catch. hmm so it shouldn't make a difference for this case since we are only direct matching for `"Groceries / Shopping"` but i'll revert it anyway!